examples/customresources/demos/simplemodule/module.go

```diff
@@ -47,11 +48,16 @@ func mainWithArgs(ctx context.Context, args []string, logger logging.Logger) err
 		return err
 	}
 
-	// This will block (leaving the module running) until the context is cancelled.
-	// The utils.ContextualMain catches OS signals and will cancel our context for us when one is sent for shutdown/termination.
-	<-ctx.Done()
-	// The deferred myMod.Close() will now run.
-	return nil
+	for {
+		if err := ctx.Err(); err != nil {
+			return nil
+		}
+		time.Sleep(time.Second)
+		logger.Debug("debug message")
+		logger.Info("info message")
+		logger.Warn("warn message")
+		logger.Error("error message")
+	}
```

Comment:
Logs above would look like:
```
2023-12-11T18:03:04.797-0500    INFO    robot_server    server/server.go:413    simple-module: info message
2023-12-11T18:03:04.797-0500    WARN    robot_server    server/server.go:415    simple-module: warn message
2023-12-11T18:03:04.797-0500    ERROR   robot_server    server/server.go:417    simple-module: error message
```
When received by a (non-debug) server.

And like:
```
2023-12-11T18:15:13.928-0500	INFO	robot_server.process.SimpleModule_/Users/benji.rewis/Code/rdk/examples/customresources/demos/simplemodule/simplemodule.StdOut	pexec/managed_process.go:244	
\_ 2023-12-11T23:15:13.928Z	INFO	simple-module	simplemodule/module.go:57	info message
2023-12-11T18:15:13.928-0500	INFO    robot_server.process.SimpleModule_/Users/benji.rewis/Code/rdk/examples/customresources/demos/simplemodule/simplemodule.StdOut	pexec/managed_process.go:244	
\_ 2023-12-11T23:15:13.928Z	WARN	simple-module	simplemodule/module.go:58	warn message
2023-12-11T18:15:13.928-0500	INFO	robot_server.process.SimpleModule_/Users/benji.rewis/Code/rdk/examples/customresources/demos/simplemodule/simplemodule.StdOut	pexec/managed_process.go:244	
\_ 2023-12-11T23:15:13.928Z	ERROR	simple-module	simplemodule/module.go:59	error message
```

When sent to the module's STDOUT due to an inactive or unhealthy gRPC connection.

I don't love the `server/server.go:41*` pieces of the RDK logs.